### PR TITLE
Feature/cic report approval

### PR DESCRIPTION
**Pull request by sroberts2**
Added CIC Report Approval classes.

Resolves: SFA-1391
                 SFA-1393

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/api/accounts/Kind.java [hunk 1]
```diff
@@ -18,7 +18,8 @@
     FIXED_ASSETS_INVESTMENTS_NOTE("small-full-accounts-note#fixed-assets-investments"),
     EMPLOYEES_NOTE("small-full-accounts-note#employees"),
     CIC_REPORT("cic-report#cic-report"),
-    CIC34_REPORT("cic34-report#cic34-report");
+    CIC34_REPORT("cic34-report#cic34-report"),
```
Comment (macreman) [inline src/main/java/uk/gov/companieshouse/api/accounts/Kind.java]: can we change this to take out the `34` ? leaving `cic-report#cic-report`

in the reworked spec we've avoided using the form number and will treat it as data going forwards so no need t include the form number
http://ch-jenkins-rh1.internal.ch:3123/manipulate-company-data/resources/cicreport?v=latest

File: src/main/java/uk/gov/companieshouse/api/accounts/ResourceName.java [hunk 2]
```diff
@@ -17,7 +17,8 @@
     FIXED_ASSETS_INVESTMENTS("fixed-assets-investments"),
     EMPLOYEES("employees"),
     CIC_REPORT("cic-report"),
-    CIC34_REPORT("cic34-report");
+    CIC34_REPORT("cic34-report"),
```
Comment (macreman) [inline src/main/java/uk/gov/companieshouse/api/accounts/ResourceName.java]: same here - can we take out the `34` ?

File: src/main/java/uk/gov/companieshouse/api/accounts/Kind.java (hunk 1 above)
Comment (BSaunders95) [reply src/main/java/uk/gov/companieshouse/api/accounts/Kind.java]: This enum was existing from work done before the spec changes and will be removed by my 'statements' changes 👍

File: src/main/java/uk/gov/companieshouse/api/accounts/ResourceName.java (hunk 2 above)
Comment (BSaunders95) [reply src/main/java/uk/gov/companieshouse/api/accounts/ResourceName.java]: As above, this was pre-existing to Steve's changes and will be removed  shortly

**Pull request merged**